pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -298,12 +298,20 @@ func retrieveMacAddress(iface *v1.Interface) (*net.HardwareAddr, error) {
 	return nil, nil
 }
 
-// The only difference between bindings for two phases is that the first phase
-// should not require access to domain definition, hence we pass nil instead of
-// it. This means that any functions called under phase1 code path should not
-// use the domain set on the binding.
-func getPhase1Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, network *v1.Network, podInterfaceName string) (BindMechanism, error) {
-	return getPhase2Binding(vmi, iface, network, nil, podInterfaceName)
+func getPhase1Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, network *v1.Network, podInterfaceName string, launcherPID int) (BindMechanism, error) {
+	if iface.Bridge != nil {
+		return generateBridgeVMNetworkingConfigurator(vmi, iface, podInterfaceName, launcherPID)
```

Comment:
Let's use `new` instead of generate. 

Also will be helpful to explain at commit message why "VMNetworkConfigurator" is here instead of something like `newBridgeBindMechanismForVirtHandler` or the like.